### ISSUE TITLE
How to restore the ideal accuracy(mAP) by fine-tuning

### ISSUE THREAD

**Issue by Lawrencechengsjtu** [musco-ai/musco-pytorch#6]
Hi author! Thx for ur sharing!

I was just trying your iterative compression algorithms using vbmf for compressing the faster rcnn model (exactly the same code mentioned in your paper), but i found it great difficulty doing the fine-tuning work.  The more layers I compressed, the less mAP it achieved.  Finally, it is approximately 8~10 points lost, which is far below your performance.

Can u tell me how I should do the fine-tuning part better? (like dataset, lr, epoch, etc.) Or can u tell me some of your opinions in terms of it? Thank u!

**Comment by Lawrencechengsjtu** [comment]
@juliagusak

**Comment by cszer** [comment]
1. compress backbone predtrained on imagenet firstly after fine tune on imagenet , it's most computational part of every detector. I think it's more appropriate approach
